src/devtools/App.vue

```diff
@@ -31,11 +31,10 @@
       <span class="pane-name">Events</span>
       <span class="event-count" v-if="newEventCount > 0">{{ newEventCount }}</span>
     </a>
-    <a class="button refresh"
+    <a class="button refresh icon-only"
```

Comment:
Why setting icon only here?

Reply:
I removed the button that toggles the layout and that had it as well. I can change this to a specific class and remove this more utilitarian class.

Reply:
I mean, we can make the Refresh button normal (by removing this class) now that you removed the other icon.

Reply:
All right. It's been removed along with the extra class.